canonical/assets.ubuntu.com
# Update dependencies on the flask version

## Done

- update deprecated dependencies that prevents server startup

## QA

- make sure to have the latest `dotrun` before doing the PR as there are some changes that needs to be done in the `dotrun`
- run the command `dotrun`
- the server should starts without errors
- you should be able to access http://localhost:8017/v1/ (with the error: missing token)
